brands/amenity/vending_machine.json

```diff
@@ -168,5 +168,47 @@
       "name": "Ключ здоровья",
       "vending": "water"
     }
+  },
+  "amenity/vending_machine|コカ・コーラ": {
+    "countryCodes": ["jp"],
+    "match": ["shop/vending_machine|コカコーラ"],
+    "nocount": true,
+    "tags": {
+      "amenity": "vending_machine",
+      "brand": "コカ・コーラ",
+      "brand:en": "Coca-Cola",
+      "brand:ja": "コカ・コーラ",
+      "drink": "cola",
+      "name": "コカ・コーラ",
+      "name:en": "Coca-Cola",
+      "name:ja": "コカ・コーラ",
+      "operator": "Nippon Coca Cola Co., Ltd.",
+      "operator:wikidata": "Q11506438",
+      "operator:wikipedia": "ja:日本コカ・コーラ",
+      "vending": "drinks",
+      "wikidata": "Q2813",
+      "wikipedia": "ja:コカ・コーラ"
+    }
+  },
+  "amenity/vending_machine|ポッカサッポロ": {
+    "countryCodes": ["jp"],
+    "match": [
+      "shop/vending_machine|POKKA SAPPORO"
+    ],
+    "nocount": true,
+    "tags": {
+      "amenity": "vending_machine",
+      "brand": "ポッカサッポロ",
+      "brand:en": "Pokka Sapporo",
+      "brand:ja": "ポッカサッポロ",
+      "brand:wikidata": "Q7208665",
+      "brand:wikipedia": "ja:Q7208665",
+      "name": "ポッカサッポロ",
+      "name:en": "Pokka Sapporo",
+      "name:ja": "ポッカサッポロ",
+      "operator": "ポッカサッポロフード&ビバレッジ",
+      "operator:en": "Pokka Sapporo Food & Beverage",
```

Comment:
If this really is the operator of all the vending machines, you can add `operator:wikidata` also

Reply:
I deleted the operator beacuse the wikipedia article is the same what the brand:wikipedia. I do not know if it is a right decision.